src/py/flwr/server/app.py

```diff
@@ -308,18 +354,33 @@ def run_server() -> None:
 
     # Start Fleet API
     if args.fleet_api_type == "rest":
-        fleet_thread = threading.Thread(
-            target=_run_fleet_api_rest,
-            args=(args.rest_fleet_api_address, state_factory),
-        )
-        fleet_thread.start()
-        bckg_threads.append(fleet_thread)
+        parsed_address = parse_address(args.rest_fleet_api_address)
+        if parsed_address:
```

Comment:
We can reduce the size of the diff here quite a bit by using `if not parsed_address: sys.exit(...)`

Reply:
... and we should use the same pattern consistently in all places where we have the address check

Reply:
It looks way better indeed! Thanks for the suggestion